web/packages/shared/components/FileTransfer/FileTransferStateless/CommonElements.tsx

```diff
@@ -56,7 +56,7 @@ export const PathInput = forwardRef<
 
 const StyledFieldInput = styled(FieldInput)`
   input {
-    border: 1px solid rgba(255, 255, 255, 0.1);
+    border: 1px solid ${props => props.theme.colors.text.placeholder};
```

Comment:
i think after @rudream changes, this will be called `muted` instead of `placeholder` right?

Reply:
Correct